src/service/poc_mobile.proto

```diff
@@ -115,6 +115,8 @@ message speedtest {
   uint64 timestamp = 4;
 }
 
+/// Deprecated reward message for mobile. See mobile_reward_share and
```

Comment:
Actually declare as deprecated.. see https://stackoverflow.com/questions/52781727/how-can-i-deprecate-whole-message-in-protocol-buffers

Reply:
good catch! Thanks